pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, gfortran, perl, liblapack, config }:
+{ stdenv, fetchurl, gfortran, perl, liblapack, config, coreutils, clang }:
```

Comment:
The clang compiler is already available through the stdenv. You only need to add it to build inputs if you're compiling something which links against its libraries (I think). So the best approach here is definitely to just omit the clang argument.